pkg/kp/kv.go

```diff
@@ -171,3 +222,16 @@ func (c consulStore) Ping() error {
 	}
 	return nil
 }
+
+func addTimeStamp(value WatchResult) (time.Time, WatchResult) {
+	currentTime := time.Now()
+	if value.Time == "" {
+		value.Time = currentTime.String()
+	}
+
+	return currentTime, value
+}
+
+func healthPath(service, node string) string {
+	return fmt.Sprintf("%s%s/%s", service, "health", node)
```

Comment:
Yeah, sorry just noticing this too. I think it's probably a better idea to have a separate part of the tree instead of concatenating the words together. `/health/{service}/{node}` is probably a good bet because it matches the pattern of having namespaces for types of data, ie `/intent`, `/reality`. As written this would make it very hard to execute a sensible watch against the health of all services.